src/assets/styles/other/_tn-styles.scss

```diff
@@ -1051,7 +1047,7 @@ $primary-dark: darken(map-get($md-primary, 500), 8%);
   // Bring tooltips to the front in dialogs
   .mat-dialog-container,
   .mat-dialog-content {
-    overflow: visible !important;
```

Comment:
This is again change at a larger scale and might result in some dialog being too short. There's a diff between `visible` and `auto`. `auto` is closer to `scroll`. This will affect all the dialog styles which is out of the scope of this ticket. We should have a specific selector.
![image](https://user-images.githubusercontent.com/24204846/151130489-462e9221-7373-40a4-9338-a4c1338c8047.png)